### PR TITLE
Accept both github 'closing' keywords and waffleboard conventions when linking to issues

### PR DESCRIPTION
**Pull request by lekkas**
Change-Type: patch
Closes: #130 

By the way, I'm totally open on simplifying the regexp to include the exact keywords, even if that means repeating the shared stems of the words cc @hedss

### CODE REVIEW

File: lib/bots/notifybot.ts [hunk 1]
```diff
@@ -85,7 +85,9 @@ interface TopicIssues {
 
 const NotifyBotPort = 8399; // Not a listed registered port
 
-const ConnectRE = /connects-to:[\s]+(#[0-9]+)/i;
+// https://help.github.com/articles/closing-issues-via-commit-messages/
+// https://help.waffle.io/faq/waffle-workflow/use-waffles-connect-keyword-to-connect-prs-to-issues
+const IssueReferenceRE = /(?:close[sd]?:?|fix(?:e[sd]{1})?:?|resolve[sd]?:?|connect(?:s|ed)?[\s]+(?:to)?)[\s]+(.*)/i;
```
Comment (hedss) [inline lib/bots/notifybot.ts]: I think Github actually requires the '#' to reference this issues, so it needs to be tested for, and in fact NotifyBot differentiates HQ from local issues by the '#', so this RegExp needs to capture the '#' as well.
Length of the RegExp doesn't bother me. :)

**Comment by hedss** [comment]
@lekkas Ping! :)

File: lib/bots/notifybot.ts (hunk 1 above)
Comment (lekkas) [inline lib/bots/notifybot.ts]: @hedss adding '#' since NotifyBot needs it.

**Comment by lekkas** [comment]
@hedss I guess we'd better go with a `rebase and merge` since this is on a already WIP branch right?

**Comment by hedss** [comment]
@lekkas Yep, that works for me! I don't think I've done anything on it since, but should be fine. :+1:

**Comment by lekkas** [comment]
cool, yeah mostly to avoid having a merge commit in your branch ;)

**Comment by hedss** [comment]
:D Thank you!

**Pull request merged**